protobuf/flow/access/access.proto

```diff
@@ -128,32 +128,39 @@ message PingResponse {}
 
 message GetLatestBlockHeaderRequest {
   bool is_sealed = 1;
+  optional bool full_block_header = 2;
 }
 
 message GetBlockHeaderByIDRequest {
   bytes id = 1;
+  optional bool full_block_header = 2;
 }
 
 message GetBlockHeaderByHeightRequest {
   uint64 height = 1;
+  optional bool full_block_header = 2;
 }
 
 message BlockHeaderResponse {
   entities.BlockHeader block = 1;
+  optional bool full_block_header = 2;
 }
```

Comment:
Do we actually need this for block header? I thought the size increase was only relevant for the full block?

Reply:
I did it to cut down on `ParentVoterIDs` and `ParentVoterSigData` that used to just be a composite object

Reply:
actually nver mind, you're right, even the signer ID size doesn't baloon too much, and we don't include the block heder in the light version of the `block` response anyway